### PR TITLE
Explicit actors in variables and methods

### PR DESCRIPTION
**Pull request by leomassazza**
- [x] Rename `debtDist` in all objects to reflect its actors (`poolsDebtDist`, `vaultsDebtDist`, etc)
- [x] Rename all debt distribution node connecting functions to `distributeDebtTo<ConnectedNodeName>()` and `adjust<ConnectedNodeName>Shares()` (keep `rebalance()`)
- [ ] Rename VaultEpoch collateralDist and consolidatedDebtList after spliting Distributions in wrappers  (TBD)

### CODE REVIEW

File: packages/synthetix-main/contracts/modules/core/MarketManagerModule.sol [hunk 1]
```diff
@@ -58,7 +58,7 @@ contract MarketManagerModule is IMarketManagerModule {
     function getMarketDebtPerShare(uint128 marketId) external override returns (int) {
         Market.Data storage market = Market.load(marketId);
 
-        market.distributeDebt(999999999);
+        market.distributeDebtToPool(999999999);
```
Comment (ajsantander) [inline packages/synthetix-main/contracts/modules/core/MarketManagerModule.sol:61]: `distributeDebtToPool<s>`

File: packages/synthetix-main/contracts/modules/core/PoolModule.sol [hunk 2]
```diff
@@ -95,7 +95,7 @@ contract PoolModule is IPoolModule {
 
         // TODO: this is not super efficient. we only call this to gather the debt accumulated from deployed pools
         // would be better if we could eliminate the call at the end somehow
-        pool.distributeDebt();
+        pool.distributeDebtToVault();
```
Comment (ajsantander) [inline packages/synthetix-main/contracts/modules/core/PoolModule.sol:98]: `distributeDebtToVault*s*()`

File: packages/synthetix-main/contracts/modules/core/MarketManagerModule.sol (hunk 1 above)
Comment (leomassazza) [reply packages/synthetix-main/contracts/modules/core/MarketManagerModule.sol]: Updated

File: packages/synthetix-main/contracts/modules/core/PoolModule.sol (hunk 2 above)
Comment (leomassazza) [reply packages/synthetix-main/contracts/modules/core/PoolModule.sol]: Updated

**Review by ajsantander — APPROVED** [review]

**Pull request merged**